CHANGELOG.md

```diff
@@ -146,6 +146,7 @@
 - The `INTERNAL_WCWIDTH` build option has been removed, as fish now always uses an internal `wcwidth` function. It has a number of configuration options that make it more suitable for general use (#5777).
 - mandoc can now be used to format the output from `--help` if `nroff` is not installed, reducing the number of external dependencies on systems with `mandoc` installed (#5489).
 - Some bugs preventing building on Solaris-derived systems such as Illumos were fixed (#5458, #5461, #5611).
+- Vendored configuration like completions and functions are sourced first from `{/usr/local,/usr}/share/fish/vendor_{completions,conf,functions}.d` and second from `$PREFIX/share/fish/vendor_{completions,conf,functions}.d` . Previously, only the latter was used. The former paths can be customized using the CMake variable `EXTRA_VENDOR_PATH`; set it to an empty string to restore the old behavior (#5029).
```

Comment:
Rather than "vendored", this should be "Third-party" or similar... though really it's probably fifth-party!